libs/ui/src/lib/modal/modal.component.ts

```diff
@@ -17,6 +17,7 @@ export class ModalComponent implements OnDestroy {
   templateRef!: TemplateRef<any>;
 
   title!: string;
+  closeable!: boolean;
```

Comment:
Should the prop have a default value?

Reply:
I'll put it to true so those that are using it directly wouldn't have to change anything to have same function they have right now